main/src/library/Vector.flix

```diff
@@ -1259,6 +1259,16 @@ mod Vector {
     pub def toSet(v: Vector[a]): Set[a] with Order[a] =
         foldRight(Set.insert, Set.empty(), v)
 
+    ///
+    /// Splits the vector `v` at position `n` into two parts
+    ///
+    pub def splitAt(n: Int32, v: Vector[a]): (Vector[a], Vector[a]) = region rc {
+        let arr = unchecked_cast(v as Array[a, rc]);
```

Comment:
We should never be using unchecked casts. It should be possible to implement the same functionality without :)